frontend/Components/Admin/Settings/LogoForm/styles.css

```diff
@@ -0,0 +1,42 @@
+ @value borderColor from 'SharedStyles/globalStyles.css';
+
+.logoFormWrapper {
+  padding: 0 10px 10px 10px;
+  border: 1px solid borderColor;
+}
+
+.logoFormHeader {
+  padding: 10px 10px 10px 0;
+}
+
+/* TODO: Reduce duplicate code in styles, a few of these are copy-pasted */
```

Comment:
Trying to figure out what the best way to re-use styles is for things like the inputs. I think maybe I should create a re-usable `Input` component like our current `Button`. Thoughts?